.vscode/launch.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Not sure we want to commit this. What's the idea?

Reply:
It allows you to launch this extension in Theia for debugging from within VSCodes launch panel. It's from the Theia extension generator. I think the goal of theia extension development is we can just open the root of the individual project and use either the launch config or ```yarn run start``` under the browser-app or electron-app directories to get fully functional theia with all the extensions we need installed. So its basically there so you don't have to use ```yarn run start``` to launch Theia with this plugin everytime you can just click ```Start browser backend``` and also to give access to debugging the backend.